src/main/fc/tasks.c

```diff
@@ -266,6 +267,13 @@ static void taskCameraControl(uint32_t currentTime)
 }
 #endif
 
+#ifdef USE_CRSF_V3
+static void taskSpeedNegotiation(uint32_t currentTime)
```

Comment:
You don't actually need this wrapper here - you can just call `speedNegotiationProcess()` as your task function.